styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
+  text-align: justify;
+  padding-top: 50px;
+  padding-bottom: 50px;
```

Comment:
ok, will add a footer. 
what other things do i need for this to flow better / to look real?
i want to give this a good skeleton and then make it look pretty...